ui/src/pages/search/authors/index.tsx

```diff
@@ -0,0 +1,315 @@
+                                    ref={searchInputRef}
+                                    name="query"
+                                    id="query"
+                                    defaultValue={props.query ? props.query : ''}
+                                    type="text"
+                                    placeholder="Type here and press enter..."
+                                    className="w-full rounded-md border border-grey-200 px-4 py-2 outline-none focus:ring-2 focus:ring-yellow-500 disabled:opacity-70"
+                                    disabled={isValidating}
+                                />
+                                <button
+                                    type="submit"
+                                    form="query-form"
+                                    aria-label="Search"
+                                    className="absolute right-px rounded-md p-2 outline-none focus:ring-2 focus:ring-yellow-500 disabled:opacity-70"
+                                    disabled={isValidating}
+                                >
+                                    <SolidIcons.SearchIcon className="h-6 w-6 text-teal-500" />
+                                </button>
+                            </label>
+                        </form>
+                    </fieldset>
+
+                    <article className="col-span-9 min-h-screen">
+                        {props.error ? (
+                            <Components.Alert
+                                severity="ERROR"
+                                title={props.error}
+                                details={['Placeholder support text here']}
+                            />
+                        ) : (
+                            <Framer.AnimatePresence>
+                                {error && (
+                                    <Components.Alert
+                                        severity="ERROR"
+                                        title={error}
+                                        details={['Placeholder support text here']}
+                                    />
+                                )}
+
+                                {!error && !results?.data?.length && !isValidating && (
+                                    <Components.Alert
+                                        severity="INFO"
+                                        title="No results found"
+                                        details={[
+                                            'Try a different search criteria.',
+                                            'If you think something is wrong, please contact the helpdesk.'
+                                        ]}
+                                    />
+                                )}
+
+                                {!isValidating && results?.data?.length && (
+                                    <>
+                                        <div className="rounded">
```

Comment:
Search results are not aligned properly. They should extend to 12 columns for now. Until we decide if and what filters to add for authors, on the left hand side, like we have for publications.
![image](https://user-images.githubusercontent.com/48569671/209105690-5a6a6551-ccdd-4e8b-8167-235048c5aa09.png)